src/components/send-button/index.js

```diff
@@ -117,31 +118,39 @@ export default compose( [
 			publishStatus = 'publish';
 		}
 
+		const [ adLabel, setAdLabel ] = useState();
+		const [ adsWarning, setAdsWarning ] = useState();
+		const [ activeAdManageUrl, setActiveAdManageUrl ] = useState();
+		const newspack_ad_post_type_slug = NEWSLETTER_AD_CPT_SLUG;
+
 		let modalSubmitLabel;
 		if ( 'manual' === serviceProviderName ) {
 			modalSubmitLabel = is_public
-				? __( 'Mark as sent and publish', 'newspack-newsletters' )
+				? __( 'Mark as send and publish', 'newspack-newsletters' )
```

Comment:
"sent" was intentional - in manual mode, the newsletter is not actually sent (that's for the user to do), so all that's available is marking as sent.

Reply:
Huh. I honestly don't remember making this change (or why I would). Will correct with a github suggestion.
```suggestion
				? __( 'Mark as sent and publish', 'newspack-newsletters' )
```